### ISSUE TITLE
program of vehicle-detection

### ISSUE THREAD

**Issue by rajabi1372** [subodh-malgonde/vehicle-detection#8]
The program didnt display an error, but I dont know why the portion of bounding box didnt run?
please help me.
[Vehicle_Detection.zip](https://github.com/subodh-malgonde/vehicle-detection/files/2716246/Vehicle_Detection.zip)
[screenshots.zip](https://github.com/subodh-malgonde/vehicle-detection/files/2716250/screenshots.zip)

**Comment by rajabi1372** [comment]
[question.zip](https://github.com/subodh-malgonde/vehicle-detection/files/2716480/question.zip)

**Comment by subodh-malgonde** [comment]
It is hard to say why the boxes don't show up. Have you made any changes to the code in the notebook? Have you loaded the weights of YOLO network into the model?

Also, what is your version of Keras and Tensorflow? The versions that I have used are more than 2 years old now. The versions which I have used are in [environment.yml](https://github.com/subodh-malgonde/vehicle-detection/blob/master/environment.yml)
